<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>601</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Podaj tekst:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>METODA TRANSPOZYCJI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj klucz:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="klucz_transpozycja">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="szyfruj_transpozycja">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Szyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="odszyfruj_transpozycja">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Odszyfruj</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>METODA MONOALFABETYCZNA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj s≈Çowo klucz:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="klucz__monoalfabet">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="odszyfruj__monoalfabet">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Odszyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="szyfruj_monoalfabet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Szyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj z pliku</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
